src/deepforest/predict.py

```diff
@@ -259,3 +273,34 @@ def _predict_crop_model_(crop_model,
     results[score_column] = score
 
     return results
+
+
+def _crop_models_wrapper_(crop_models,
+                          trainer,
+                          results,
+                          path,
+                          transform=None,
+                          augment=False):
+    if crop_models is not None and not isinstance(crop_models, list):
+        crop_models = [crop_models]
+
+    # Run predictions
+    crop_results = []
+    if crop_models:
+        is_single_model = len(
+            crop_models) == 1  # Flag to check if only one model is passed
+        for i, crop_model in enumerate(crop_models):
+            crop_result = _predict_crop_model_(crop_model=crop_model,
+                                               results=results,
+                                               path=path,
+                                               trainer=trainer,
+                                               model_index=i,
+                                               transform=transform,
+                                               augment=augment,
+                                               is_single_model=is_single_model)
+            crop_results.append(crop_result)
```

Comment:
Not sure if it's worth addressing this now, but since this is the second time this construct has come up in this PR:

It seems like it would be better to let `_predict_crop_model_` take a list for `crop_model` and then handle checking if length > 1, generating model indices, and concatenating the results.